components/Workspaces/WorkspaceWelcomeModal.tsx

```diff
@@ -43,11 +43,17 @@ export default function WorkspaceWelcomeModal({ isOpen, onClose }: WorkspaceWelc
       title={TITLE}
       description={
         <div className="text-center flex flex-col items-center justify-between gap-4 !p-8">
-          <img
-            src="/assets/workspace_overview.png"
-            alt="Workspace screenshot from documentation"
+          <iframe
+            width="100%"
+            height="300px"
```

Comment:
Refactor (blocking): [lazy load the  iframe](https://web.dev/articles/iframe-lazy-loading).

```suggestion
          <iframe
            width="100%"
            height="300px"
            loading="lazy"
```